code/datums/gamemode/factions/bloodcult/bloodcult_spells.dm

```diff
@@ -10,7 +10,7 @@
 /*
 /spell/cult/trace_rune
 	name = "Trace Rune"
-	desc = "(1 BLOOD) Use available blood to write down words. Three words form a rune."
+	desc = "(2 BLOOD) Use available blood to write down words. Three words form a rune."
```

Comment:
This has been commented out man, you didn't change a thing, this is still unused.

Reply:
i still got to update the pr without the blood cost changes  but this was changed so if anyone wanted to use the spell code insted of the mindUI.

Reply:
You need to change it back because you haven't actually done the blood cost change anywhere, this will just be a confusing artifact in comments

Reply:
there we got it's back to how it used to be